src/sentry/sentry_metrics/indexer/indexer_consumer.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from typing import Any, Dict
+
+from django.conf import settings
+
+from sentry.sentry_metrics import indexer
+from sentry.sentry_metrics.indexer.base import UseCase
+from sentry.utils import json, kafka_config
```

Comment:
Please ensure you use rapidjson for parsing and serializing json

https://github.com/getsentry/sentry/blob/master/src/sentry/utils/json.py#L98-L115